.cirrus.yml

```
@@ -0,0 +1,119 @@
+env:
+  CPPFLAGS: -I/usr/local/include
+  LDFLAGS: -L/usr/local/lib
+  PREFIX: ${CIRRUS_WORKING_DIR}/project
+  PATH: ${PREFIX}/bin:${PATH}
+  BIRCH_FLAGS: '--unit=dir --arch=native --jobs=2'
+  MODE_FLAGS: '--disable-debug --enable-test --enable-release'
+  OMP_NUM_THREADS: 2
+
+task:
+  name: FreeBSD
+  freebsd_instance:
+    image_family: freebsd-12-1
+    cpu: 2
+    memory: 4G
+  install_script:
+    - pkg install -y automake bash bison boost-libs cairo eigen libtool libyaml sqlite3
+  driver_build_script:
+    - cd driver
+    - ./bootstrap
+    - ./configure --prefix="${PREFIX}" CXXFLAGS="-Wall -Wno-overloaded-virtual -g -O0 -fno-inline --coverage" || cat config.log
+    - make -j4
+    - make install
+  driver_test_script:
+    - mkdir hello
+    - cd hello
+    - birch help
+    - birch init
+    - birch audit
+    - birch dist
+    - birch clean
+  libbirch_build_script:
+    - cd libbirch
+    - ./bootstrap
+    - ./configure --prefix="${PREFIX}" ${MODE_FLAGS} || cat config.log
+    - make -j4
+    - make install
+  birch-standard_build_script:
+    - cd libraries/Standard
+    - birch build --prefix="${PREFIX}" ${MODE_FLAGS} ${BIRCH_FLAGS}
+    - birch install
+  birch-standard_smoke_script:
+    - cd libraries/Standard
+    - env BIRCH_MODE=test bash ./smoke.sh
```

```suggestion
    - bash ./smoke.sh
```
If recommendation above is accepted to only build debug mode, we now need to specify debug mode here rather than test mode, as we didn't build test. As debug is the default, the `BIRCH_MODE` environment variable can be omitted.